moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -162,11 +162,13 @@ struct CollisionResult
   /** \brief Number of contacts returned */
   std::size_t contact_count;
 
-  /** \brief A map returning the pairs of ids of the bodies in contact, plus information about the contacts themselves
+  /** \brief Mapping of the pairs of bodies in contact, plus information about the contacts themselves.
+   *
+   *  Each contact pair can consist of multiple contacts.
```

Comment:
I don't think I understand; this seems to have become less clear. I assume you mean "Each body pair can include multiple contacts"?

Reply:
Yes, that is what I meant. Thanks!